daemon/api_interfaces_test.go

```diff
@@ -1200,7 +1200,7 @@ func (s *interfacesSuite) TestInterfacesAllDefaultDocURL(c *check.C) {
 	c.Check(err, check.IsNil)
 
 	if result, ok := body["result"].([]interface{}); ok {
-		c.Assert(len(result), check.Equals, 190, check.Commentf("New interface detected, please ensure it is documented at https://snapcraft.io/docs/<name>-interface and increment the count"))
+		c.Assert(len(result), check.Equals, builtin.BuiltinIfaceCount, check.Commentf("New interface detected, please ensure it is documented at https://snapcraft.io/docs/<name>-interface and increment BuiltinIfaceCount"))
```

Comment:
if you really want to have something like this, it should be  test failing in interfaces/builtin, not in daemon. People working on interfaces are to always likely to run daemon tests